### PR TITLE
Feature/annotations tests

### PR DESCRIPTION
**Pull request by picciama**
This adds a bunch of tests to increase code coverage in various input scenarios.

**PR Checklist**

-   [x] This comment contains a description of changes (with reason)
-   [x] Referenced issue is linked
-   [x] If you've fixed a bug or added code that should be tested, add tests!
-   [x] Documentation in `docs` is updated

**Description of changes**

Doesn't affect the user. Only unit tests that cover most of the package now.
**Technical details**

**Additional context**

### CODE REVIEW

File: spectrum_fundamentals/annotation/annotation.py [hunk 1]
```diff
@@ -212,7 +241,7 @@ def parallel_annotate(spectrum: pd.Series, index_columns: dict):
     if len(matched_peaks) == 0:
         intensity = np.full(174, 0.0)
         mass = np.full(174, 0.0)
-        return intensity, mass, calc_mass
+        return intensity, mass, calc_mass, 0
```
Comment (WassimG) [inline spectrum_fundamentals/annotation/annotation.py:244]: I don't get why did you add a zero here to the return statement and how useful is this if it is a constant value.

Comment (picciama) [reply spectrum_fundamentals/annotation/annotation.py:244]: because it should return only one type of tuple!

**Review by WassimG — APPROVED** [review]
